standalone/detectors/fasttrack/include/PoolAllocator.h

```diff
@@ -0,0 +1,199 @@
+#ifndef POOL_ALLOCATOR_HEADER_H
+#define POOL_ALLOCATOR_HEADER_H 1
+#pragma once
```

Comment:
Please remove insignificant `1` and the redundant pragma since include guards already exist and are used 
```suggestion
#define POOL_ALLOCATOR_HEADER_H
```